main.go

```diff
@@ -107,11 +114,15 @@ func main() {
 		go statusService(statusAddr)
 	}
 
-	uri, err := url.Parse(fmt.Sprintf("relay://%s/?id=%s", extAddress, id))
+	uri, err := url.Parse(fmt.Sprintf("relay://%s/?id=%s&pingInterval=%s&networkTimeout=%s&sessionLimitBps=%d&globalLimitBps=%d&statusAddr=%s", extAddress, id, pingInterval, networkTimeout, sessionLimitBps, globalLimitBps, statusAddr))
```

Comment:
Depends on the view, from my view it's not a bunch of values, it's a single value posted, and the value is the DSN of the relay server which provides all the information the client might need to connect (or decide whether to connect)

This does not get expanded on the relaypool, and stays as:
```
{'relays':[
  {'url': 'something.com/?pingInterval10'}
]}
```
when the client looks it up.

It's just metadata about the relay in it's DSN, so that clients could decide if it's the right one for them, or not.

Yes, we could post JSON, but then the client can't just accept a string which has everything in it. There will probably be 0 or perhaps maximum 1 place in the code where these fields will actually be used, and I am ready to make the sacrifice of doing an unsafe uri.Get('pingInterval') instead of throwing a struct around all over the place, with fields that might never even be used.

Plus, this allows superb flexibility, where we can add random arbitrary pieces of information embedded in the URL without breaking backwards compatibility or requiring users to map the new field in the struct.